src/site/paragraphs/react_widget.drupal.liquid

```diff
@@ -40,7 +40,11 @@
       {% if entityUrl.path contains "/eligibility" %}data-widget-show-learn-more{% endif %}
       data-widget-timeout="{{entity.fieldTimeout}}">
       <div class="loading-indicator-container">
-        <div class="loading-indicator" role="progressbar" aria-valuetext="{{ entity.fieldLoadingMessage }}"></div>
+        <div
+          aria-label="{% if entity.fieldLoadingMessage %}{{ entity.fieldLoadingMessage }}{% else %}loading indicator{% endif %}"
+          class="loading-indicator"
```

Comment:
I'd also like to add an `aria-valuetext` line to offer max screen reader usability. We tested this heavily with JAWS, NVDA, VO, and came up with this fallback text in case the screen reader didn't announce the label:

```html
aria-valuetext="Loading your application..."
```